connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/processors/mbop/MBOPRequestPreparer.java

```diff
@@ -0,0 +1,88 @@
+package com.redhat.cloud.notifications.connector.email.processors.mbop;
+
+import com.redhat.cloud.notifications.connector.email.config.FeatureFlipper;
+import com.redhat.cloud.notifications.connector.email.constants.ExchangeProperty;
+import com.redhat.cloud.notifications.connector.email.model.User;
+import com.redhat.cloud.notifications.connector.email.model.mbop.Email;
+import com.redhat.cloud.notifications.connector.email.model.mbop.Emails;
+import io.vertx.core.json.JsonObject;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.apache.camel.component.http.HttpMethods;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@ApplicationScoped
+public class MBOPRequestPreparer implements Processor {
```

Comment:
We'll talk to BOP most of the time, MBOP is an exception in some environments. The package and class could be renamed to BOP :smiley:

Reply:
True. Renamed!